browser-sign-in-and-biometric-storage/OktaBrowserSignIn/Okta.plist

```diff
@@ -5,7 +5,7 @@
 	<key>scopes</key>
 	<string>openid profile offline_access</string>
 	<key>redirectUri</key>
-	<string>{customUriScheme}:/callback</string>
+	<string>{customUriScheme}:/loginCallback</string>
```

Comment:
👍

Reply:
🙌  Just to make consistent between iOS and react-native sample.